Client/Frontend/Library/LibraryViewController/LibraryViewController.swift

```diff
@@ -217,22 +231,110 @@ class LibraryViewController: UIViewController {
         }
     }
 
+    func updateViewWithState() {
+        updateBookmarkPanelState()
+        topLeftButtonSetup()
+        topRightButtonSetup()
+    }
+
+    func updateBookmarkPanelState() {
+        if librarySegmentControl.selectedSegmentIndex == 0 {
+            if let panel = children.first as? UINavigationController,
+               panel.viewControllers.count > 1 {
+                if bookmarkPanelState == .home {
```

Comment:
How do you feel about using guards or cleaning nested if lets? 🤔 
^maybe for later

Reply:
I had `guard let` there at one point. I think I just accidentally undid it. I can change that.